src/libgv/strcode.c

```diff
@@ -1,5 +1,11 @@
 #include <SYS/TYPES.H>
 
+/**
+ * @brief Generate a simple hash code from a string.
+ *
+ * @param string input string
+ * @return int 32-bit hash
```

Comment:
GV_StrCode returns a 32-bit integer but the hash itself is only 16-bits. (Same thing with the MGS2 version where the hash is masked to 24-bits.)